### DISCUSSION TITLE
Replacing ls output with exa output

### DISCUSSION THREAD

**Discussion by zakissimo** [gokcehan/lf#788] [Q&A] [answered]
Is it possible to replace ls output by exa ?

Current :
![image](https://user-images.githubusercontent.com/64891486/158360441-e409b745-6bee-4cbc-a7e1-b91ea7b25800.png)

Desired:
![image](https://user-images.githubusercontent.com/64891486/158360617-a5f53f3f-e1a3-40f3-b009-4a881f0bf21b.png)

**Comment by gokcehan** [comment]
@zakissimo There's `timefmt` option you can use to be able to mimic that style to some extent I think.

**Reply by zakissimo** [reply]
I don't really mind the format, I just want the pretty colors. That being said, I know nothing about go :/ , I tried live greping for "ls" it the git repo but didn't find something relevent.

**Reply by CatsDeservePets** [reply, marked as answer]
Hey @zakissimo,

this is now finally possible!
Thanks to https://github.com/gokcehan/lf/pull/2186!
The discussion we had was about multiple issues and posts. I am not able to quickly find it but if I remember correctly @joelim-work even had an example for this exact use case.

Edit: Nevermind, I found it. https://github.com/gokcehan/lf/pull/2083
